src/palindrome.js

```diff
@@ -0,0 +1,13 @@
+export default function isPalindrome(str){
+  let reversedStr = str
+  reversedStr = reversedStr.replace(/[^a-zA-z]/g, '')
```

Comment:
Maybe the regular expression should be `/[^a-zA-Z]/g`?  It seems to be working nonetheless.